waspc/data/Generator/templates/react-app/vitest.setup.ts

```diff
@@ -0,0 +1,8 @@
+import matchers from '@testing-library/jest-dom/matchers'
```

Comment:
Is this setup file used implicitly?

Reply:
Another question, can we configure `vitest` to maybe reduce the amount of output on each test run?
<img width="850" alt="Screenshot 2023-03-23 at 14 08 28" src="https://user-images.githubusercontent.com/2223680/227214031-c5aacb0d-2991-4f2b-9dff-667bb07339ed.png">

Reply:
This file is referenced by `vite.config.ts`.

That output is from `screen.debug()`, mainly just for showing the render works. Most users won't leave that in there.